bin/key.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "service_account",
```

Comment:
as per discussion, encrypt this with gcloud kms encrypt

Reply:
looking at that